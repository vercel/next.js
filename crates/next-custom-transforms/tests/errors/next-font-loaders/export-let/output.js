import React from 'react';
import firaCode from '@next/font/google/target.css?{"path":"pages/test.tsx","import":"Abel","arguments":[],"variableName":"firaCode"}';
import inter from '@next/font/google/target.css?{"path":"pages/test.tsx","import":"Inter","arguments":[],"variableName":"inter"}';
export { firaCode };
export { inter };
