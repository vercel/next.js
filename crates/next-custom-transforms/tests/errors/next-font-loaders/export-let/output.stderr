  x Font loader calls must be assigned to a const
   ,-[input.js:4:1]
 3 | 
 4 | export let firaCode = Abel()
   :        ^^^^^^^^^^^^^^^^^^^^^
 5 | export var inter = Inter()
   `----
  x Font loader calls must be assigned to a const
   ,-[input.js:5:1]
 4 | export let firaCode = Abel()
 5 | export var inter = Inter()
   :        ^^^^^^^^^^^^^^^^^^^
   `----
